lib/activerecord-import/import.rb

```diff
@@ -123,6 +123,15 @@ def supports_on_duplicate_key_update?
       false
     end
 
+    # returns true if the current database connection adapter
+    # supports setting the primary key of bulk imported models, otherwise
+    # returns false
+    def supports_setting_primary_key?
+      connection.supports_setting_primary_key?
+    rescue NoMethodError
```

Comment:
Oh what the heck: https://github.com/zdennis/activerecord-import/commit/556a0c8d1a66992be9c673347131627bb6bac829